support/rewriteInheritance.js

```diff
@@ -0,0 +1,29 @@
+var through = require('through2');
+
+/**
+ * Replace all instances of `__proto__` in a string with a cross-browser
+ * compatible alternative.
+ *
+ * @param {string} js
+ * @return {string}
+ */
+function parseInheritance(js) {
+  return js.replace(/^ *(\w+)\.prototype\.__proto__ * = *(\w+)\.prototype *;?/gm, function(_, child, parent){
```

Comment:
:/ I appreciate the effort, but we really should remove all usages of `__proto_` from the codebase. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/proto for instance.

Couldn't we apply this rewrite rule verbatim to our codebase, rather than at build-time?

```bash
git grep __proto lib
```
> lib/hook.js:Hook.prototype.__proto__ = Runnable.prototype;
lib/reporters/dot.js:Dot.prototype.__proto__ = Base.prototype;
lib/reporters/landing.js:Landing.prototype.__proto__ = Base.prototype;
lib/reporters/list.js:List.prototype.__proto__ = Base.prototype;
lib/reporters/min.js:Min.prototype.__proto__ = Base.prototype;
lib/reporters/nyan.js:NyanCat.prototype.__proto__ = Base.prototype;
lib/reporters/progress.js:Progress.prototype.__proto__ = Base.prototype;
lib/reporters/spec.js:Spec.prototype.__proto__ = Base.prototype;
lib/reporters/xunit.js:XUnit.prototype.__proto__ = Base.prototype;
lib/runnable.js:Runnable.prototype.__proto__ = EventEmitter.prototype;
lib/runner.js:Runner.prototype.__proto__ = EventEmitter.prototype;
lib/suite.js:Suite.prototype.__proto__ = EventEmitter.prototype;
lib/test.js:Test.prototype.__proto__ = Runnable.prototype;

It looks like the transform below would change
```javascript
Hook.prototype.__proto__ = Runnable.prototype;
```
into
```javascript
function F(){};
F.prototype = Runnable.prototype;
Hook.prototype = new F;
Hook.prototype.constructor = Hook;
```

Is there any reason to not just apply this to the entire codebase and remove this rewrite rule?